### PR TITLE
Additional eventing sample edits

### LINKED ISSUE #2704: Fix review comments from #2299
**Describe the change you'd like to see**
Fix review comments from #2299

**Additional context**
Usually these should be fixed by the person opening a PR but in this case I am making an exception to get the content included at least.

### PR DESCRIPTION
**Pull request by vmexalien**
Fixes #2704 

## Proposed Changes 

- enhancements to docs for consistency and style

### CODE REVIEW

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+
+    ```
+
+1. Add a requirements.txt file containing the following contents:
+
+    ```bash
+    Flask==1.1.1
+    ```
+1. In your project directory, create a file named `Dockerfile` and copy the code
+   block below into it. For detailed instructions on dockerizing a Go app, see
```
Comment (mpetason) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:58]: Maybe we should replace the "dockerizing a go app" section with something related to Python, or possibly just remove the line related to Go?

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+   ```shell
+   # Build the container on your local machine
+   docker build -t {username}/helloworld-python .
+
+   # Push the container to docker registry
+   docker push {username}/helloworld-python
+   ```
+
+1. After the build has completed and the container is pushed to docker hub, you
```
Comment (mpetason) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:162]:
```suggestion
1. After the build has completed and the container is pushed to Docker Hub, you
```

Updated for consistency.

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+        ```shell
+        kubectl --namespace knative-samples get deployments helloworld-python
+        kubectl --namespace knative-samples get svc helloworld-python
+        ```
+    1. It created a Knative Eventing Trigger to route certain events to the helloworld-python application. Make sure that Ready=true
+        ```shell
+        kubectl --namespace knative-samples get trigger helloworld-python
+        ```
+## Send and verify CloudEvents
+Once you have deployed the application and verified that the namespace, sample application and trigger are ready, let's send a CloudEvent.
```
Comment (mpetason) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:184]:
```suggestion
Once you have deployed the application and verified that the namespace, sample application, and trigger are ready, let's send a CloudEvent.
```

**Comment by mpetason** [comment]
The CLA issue happens when you accept comments instead of making changes and pushing the new commit. 

/ok-to-test

**Comment by mpetason** [comment]
The CLA issue happens when you pull in changes from someone else. I signed the CLA with a different email address associated with ibm - so co-authoring as the same user that I use to make comments breaks it.

**Comment by grantr** [comment]
I looked at CLA status. `vmexalien` and `axsaucedo` are both recorded as signing the CLA. `mpetason` is not but consented above.

> I signed the CLA with a different email address associated with ibm - so co-authoring as the same user that I use to make comments breaks it.

You can sign the CLA with your github-associated email too, then you'll be covered both ways.

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 3 above)
Comment (vmexalien) [reply docs/eventing/samples/helloworld/helloworld-python/README.md:184]: @mpetason I like what you have added for punctuation, but for consistency, 'let's' should not be used.

Better put would be - "Once you have deployed the application and verified that the namespace, sample application, and trigger are ready, a CloudEvent can be sent."

**Comment by grantr** [comment]
Hi @vmexalien! I'd be happy to help you get this merged, if you're still interested. You had some merge conflicts so I took a stab at resolving them. You can see the changes I made in this commit: https://github.com/knative/docs/pull/2930/commits/0e25c0b35bb01e1c1be6a53e24d4d62661c2b198.

@mpetason @abrennan89 This is probably ready to be reviewed.

**Comment by grantr** [comment]
@googlebot I consent.

**Comment by grantr** [comment]
All authors have consented.

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 4]
```diff
@@ -184,8 +184,7 @@ folder) you're ready to build and deploy the sample app.
 Once you have deployed the application and verified that the namespace, sample application and trigger are ready, let's send a CloudEvent.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:184]:
```suggestion
After you have deployed the application, and have verified that the namespace, sample application and trigger are ready, you can send a CloudEvent.
```
Stick with the active voice and no fluff :) https://developers.google.com/style/voice

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 5]
```diff
@@ -184,8 +184,7 @@ folder) you're ready to build and deploy the sample app.
...
 
 ### Send CloudEvent to the Broker
-
-We can send an http request directly to the Broker with correct CloudEvent headers set.
+We can send an http request directly to the [Broker](../../../broker-trigger.md) with correct CloudEvent headers set.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:187]:
```suggestion
You can send an HTTP request directly to the Knative [broker](../../../broker-trigger.md) with correct CloudEvent headers set.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 6]
```diff
@@ -235,7 +234,7 @@ Helloworld-python app logs the context and the msg of the above event, and repli
         {"msg":"Hi from Knative!"}
 
       ```
-  Play around with the CloudEvent attributes in the curl command and the trigger specification to understand how Triggers work.
+  Play around with the CloudEvent attributes in the curl command and the trigger specification to understand how [Triggers work](../../../broker-trigger.md#trigger).
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:237]:
```suggestion
  Try the CloudEvent attributes in the curl command and the trigger specification to understand how [triggers work](../../../broker-trigger.md#trigger).
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 7]
```diff
@@ -235,7 +234,7 @@ Helloworld-python app logs the context and the msg of the above event, and repli
...
 
 ## Verify reply from helloworld-python app
 `helloworld-python` app replies back with an event of `type= dev.knative.samples.hifromknative`, and `source=knative/eventing/samples/hello-world`. This event enters the eventing mesh via the Broker and can be delivered to other services using a Trigger
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:240]:
```suggestion
The `helloworld-python` app replies with an event type `type= dev.knative.samples.hifromknative`, and source `source=knative/eventing/samples/hello-world`. The event enters the eventing mesh through the broker, and can be delivered to event sinks using a trigger
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Thanks @vmexalien, just some suggestions

**Comment by abrennan89** [comment]
/lgtm

@grantr can you fix the CLA please

**Comment by abrennan89** [comment]
@googlebot I consent.

**Review by abrennan89 — APPROVED** [review]

**Comment by grantr** [comment]
All authors have (still) consented.

**Pull request merged**